src/main/java/net/minecraftforge/client/model/ModelLoaderRegistry.java

```diff
@@ -63,17 +65,23 @@
     // Forge built-in loaders
     public static void init()
     {
-        registerLoader(new ResourceLocation("minecraft","elements"), VanillaProxy.Loader.INSTANCE);
-        registerLoader(new ResourceLocation("forge","obj"), OBJLoader.INSTANCE);
-        registerLoader(new ResourceLocation("forge","bucket"), DynamicBucketModel.Loader.INSTANCE);
-        registerLoader(new ResourceLocation("forge","composite"), CompositeModel.Loader.INSTANCE);
-        registerLoader(new ResourceLocation("forge","multi-layer"), MultiLayerModel.Loader.INSTANCE);
-        registerLoader(new ResourceLocation("forge","item-layers"), ItemLayerModel.Loader.INSTANCE);
-        registerLoader(new ResourceLocation("forge", "separate-perspective"), SeparatePerspectiveModel.Loader.INSTANCE);
+        var builtInLoaders = Map.of(
+            new ResourceLocation("minecraft","elements"), VanillaProxy.Loader.INSTANCE,
+            new ResourceLocation("forge","obj"), OBJLoader.INSTANCE,
+            new ResourceLocation("forge","bucket"), DynamicBucketModel.Loader.INSTANCE,
+            new ResourceLocation("forge","composite"), CompositeModel.Loader.INSTANCE,
+            new ResourceLocation("forge","multi-layer"), MultiLayerModel.Loader.INSTANCE,
+            new ResourceLocation("forge","item-layers"), ItemLayerModel.Loader.INSTANCE,
```

Comment:
Missing space after the comma before the RL path (yes I know it was missing before as well but might as well fix it)